tools/ttysnoop.py

```diff
@@ -28,10 +28,13 @@ def usage():
 
 # arguments
 examples = """examples:
-    ./ttysnoop /dev/pts/2    # snoop output from /dev/pts/2
-    ./ttysnoop 2             # snoop output from /dev/pts/2 (shortcut)
-    ./ttysnoop /dev/console  # snoop output from the system console
-    ./ttysnoop /dev/tty0     # snoop output from /dev/tty0
+    ./ttysnoop /dev/pts/2           # snoop output from /dev/pts/2
+    ./ttysnoop 2                    # snoop output from /dev/pts/2 (shortcut)
+    ./ttysnoop /dev/console         # snoop output from the system console
+    ./ttysnoop /dev/tty0            # snoop output from /dev/tty0
+    ./ttysnoop /dev/pts/2  -s 1024  # snoop output from /dev/pts/2 with data size 1024
+    ./ttysnoop /dev/pts/2  -c 2     # snoop output from /dev/pts/2 with 2 checks for 4096 bytes of data in buffer
+                                      (potentially retrieving 8192 bytes)
```

Comment:
could you explain why '-c' option is needed? I see the default below is 20? Why choose 20?

Reply:
with most 'big bcurses' apps I'm getting 4096 as maximum for count,
so I guess it's the limit somewhere in the kernel code, I can change the default to be closer to that